script.js

```diff
@@ -0,0 +1,84 @@
+const menuBtn = document.getElementById('menu-btn-open');
+const overlay = document.getElementById('overlay');
+const menu = document.getElementById('mobile-menu');
+
+function navToggle() {
+  menuBtn.classList.toggle('open');
+  overlay.classList.toggle('overlay-show');
+  document.body.classList.toggle('stop-scrolling');
+  menu.classList.toggle('show-menu');
+}
+
+menuBtn.addEventListener('click', navToggle);
+
+document.documentElement.addEventListener('click', (e) => {
+  if (e.target.matches('.menu-item')) {
+    menu.classList.remove('show-menu');
+    menuBtn.classList.remove('open');
+    overlay.classList.remove('overlay-show');
+    document.body.classList.remove('stop-scrolling');
+  }
+});
+
+// Speaker Data
+const speakers = [
```

Comment:
Great job @MoyasiGinko 🤝

- [ ] [Optional] I would recommend you follow the given design as much as possible. Kindly implement the `more` button so that you can replicate the design given.
- Your design
![your design](https://user-images.githubusercontent.com/63917725/235011031-ca12a9af-c7f0-4cd1-8edd-617775f7ab35.PNG)
- Given design
![given design](https://user-images.githubusercontent.com/63917725/235011074-523944d3-938b-4f5a-942a-4cced34b9491.PNG)